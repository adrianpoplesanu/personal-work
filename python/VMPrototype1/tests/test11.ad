if (1 < 2) { 11 } else { 22 }