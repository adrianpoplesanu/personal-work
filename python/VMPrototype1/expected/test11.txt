ASTProgram
ASTExpressionStatement\[ASTIfExpression\]
\[0, 0, 0, 0, 0, 1, 10, 14, 0, 16, 0, 0, 2, 15, 0, 19, 0, 0, 3, 5\]
0000 OpConstant 0
0003 OpConstant 1
0006 OpGreaterThan
0007 OpJumpNotTruthy 16
0010 OpConstant 2
0013 OpJump 19
0016 OpConstant 3
0019 OpPop
11
